MinecraftClient/ConsoleIO.cs

```diff
@@ -336,6 +336,14 @@ public static void WriteLineFormatted(string str, bool acceptnewlines = true, bo
                 }
                 if (BasicIO)
                 {
+                    if (Settings.ConsoleIOColor_Enabled == false)
+                    {
+                        string colorcodes = "0123456789abcdefklmnor";
+                        foreach (char c in colorcodes)
+                        {
+                          str = str.Replace("§"+c, string.Empty);
+                        }
```

Comment:
Instead of rewriting this, you can use the existing method `GetVerbatim` from the ChatBot class [here](https://github.com/ORelio/Minecraft-Console-Client/blob/master/MinecraftClient/ChatBot.cs#L243) after switching it to Public visibility.